### PR TITLE
Update easy-way-of-sharing-files.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Removed the link with extra space from the content as it was reported by a user to have a "mistakenly places space". Since its possible it can create confustion, I moved it to the links section

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/linux/user-and-file-management/file-management-tips-ii/easy-way-of-sharing-files.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/linux/user-and-file-management/file-management-tips-ii/easy-way-of-sharing-files.md?ref=482349d4b70370f52b6912e878793c6b59175f1b)

File: linux/user-and-file-management/file-management-tips-ii/easy-way-of-sharing-files.md [hunk 1]
```diff
@@ -19,7 +23,7 @@ File transfer on the network without setting up extra software can be easily ach
 
 `woof` (Web Offer One File) is a script and will run on any machine with *Python* installed.
 
-It can be downloaded at: `www.home.unix-ag.org/ simon/woof.html`.
+You can check download it from the link in the **Learn More** section.
```
Comment (kapnobatai137) [inline linux/user-and-file-management/file-management-tips-ii/easy-way-of-sharing-files.md:26]: what do you think about this?

```suggestion
To download it, check the **Learn More** section.
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

File: linux/user-and-file-management/file-management-tips-ii/easy-way-of-sharing-files.md (hunk 1 above)
Comment (Stefan-Stojanovic) [reply linux/user-and-file-management/file-management-tips-ii/easy-way-of-sharing-files.md:26]: Uh, what happened. That was not what I wrote haha.

It was supposed to say, `you can check it out at the **Learn More** section.`

Yeah I like that I'll commit it

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**